docs/network/evm-nodes.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Running an Etherlink EVM node
+---
+
+The Etherlink EVM nodes are responsible for maintaining a copy of the Etherlink context and creating and applying new blocks that process EVM transactions.
```

Comment:
Here we explain the worflow for the observer EVM node. The observer don't create blocks,
```suggestion
The Etherlink EVM nodes are responsible for maintaining a copy of the Etherlink context and applying new blueprints that process EVM transactions.
```